cmd/collector/app/metrics.go

```diff
@@ -59,28 +59,28 @@ func initOtherServicesSamplers() map[model.SamplerType]string {
 }
 
 // SpanProcessorMetrics contains all the necessary metrics for the SpanProcessor
-type SpanProcessorMetrics struct {
```

Comment:
this class has nothing to do with the ticket. SpanMetricsProcessor refers to an OTEL Collector component.

Reply:
Yes, got it. At the time of this PR raised I thought it is as a Jaeger component. After working ony proposal for SPM, I got clear context of it.

Now, I will remove that flag and corresponding code and will make default of spanmetrics connector and add it the same in Readme